### DISCUSSION TITLE
How to add custom css for options?

### DISCUSSION THREAD

**Discussion by mostopalove** [sa-si-dev/virtual-select#170] [Q&A] [answered]
Can I add `class` or `style` attribute for `<li>` tag?
For example: set `background-color` for one element in dropdown list?

**Comment by sa-si-dev** [comment]
Use [labelRenderer](https://sa-si-dev.github.io/virtual-select/#/examples?id=add-imageicon) method to customize options.

**Reply by mostopalove** [reply]
It works only for labels but not for label container `<li>`

**Reply by mostopalove** [reply]
<img width="254" alt="image" src="https://user-images.githubusercontent.com/49098339/183292707-60e15dfc-88c2-4709-83f3-753e812e0ce7.png">

**Reply by mostopalove** [reply]
I can rewrite `vscomp-option` style and set my custom label as I want. But if you know more pretty solution I'll be happy 😄

**Reply by sa-si-dev** [reply]
As if now, there is no option to customize options `li`.

May I know what is your exact requirement?

**Reply by mostopalove** [reply]
I want to set color for some special option, like main option

**Comment by sa-si-dev** [comment, marked as answer]
@mostopalove you could set the new property `options[].classNames` to customize option element.

This feature available in the release [v1.0.32](https://github.com/sa-si-dev/virtual-select/releases/tag/v1.0.32)
